id_infrastructure/firestore_uuid_table.py

```diff
@@ -36,7 +37,7 @@ def data_to_uuid_batch(self, list_of_data_requested):
         new_mappings_needed = set_of_data_requested.difference(
             set(existing_mappings.keys()))
 
-        print ("new_mappings_needed: {}".format(len(new_mappings_needed)))
+        log.info("New mappings needed: {}".format(len(new_mappings_needed)))
```

Comment:
ok - if we're going to make a bunch of changes here, let's also standardise to f"{len...}" at the same time

Reply:
done